HW_4_Numpy/numpy_challenge.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+
+
+def matrix_multiplication(a, b):
+    """
+    Takes 2 matrices, multiplies them according to the
+    rules of the matrix product, gives the resulting matrix
+
+    :param a: 2D np.array with (x,y) shape
+    :param b: 2D np.array with (y,z) shape
+    :return: 2D np.array with (x,z) shape
+    """
+    return np.matmul(a, b)
+
+
+def multiplication_check(m_list):
```

Comment:
Не надо жалеть буквы

```suggestion
def multiplication_check(matrix_list):
```